File: Assignment_13/Pattern_3.c
///////////////////////////////////////////////////////
//
//  File name :     Pattern_3
//  Description:    Accept number of rows and columns from user and display pattern. 
//  Author :        Trupti Bhatlekar
//  Date :          25/05/2025
//
///////////////////////////////////////////////////////
// Input: 4 4 
// Output: A A A A 
//         B B B B 
//         C C C C 
//         D D D D          
///////////////////////////////////////////////////////

#include <stdio.h>

void Pattern(int iRow, int iCol)
{

  int i = 0; int j = 0;
  char ch = '\0';

  for (i = 0; i < iRow; i++)
  {
    ch = 'A' + i;

    for (j = 0; j < iCol; j++)
    {
      printf("%c\t", ch);
    }
    printf("\n");
  }
}

int main()
{
  int iValue1 = 0, iValue2 = 0;

  printf("Enter number of rows and columns:\n");
  scanf("%d %d", &iValue1, &iValue2);

  Pattern(iValue1, iValue2);

  return 0;
}

File: Assignment_46/Pattern_3.java
///////////////////////////////////////////////////////
//
//  File name :     Pattern_3
//  Description:    To accept number of rows and number of column from user and display pattern.
//  Author :        Trupti Bhatlekar
//  Date :          29/08/2025
//
///////////////////////////////////////////////////////

//////////////////////////////////////////////////////
// Input: iRow = 3, iCol = 5
// Output: A A A A A
//         B B B B B
//         C C C C C
///////////////////////////////////////////////////////

import java.util.*;

class Pattern
{
    public void Pattern(int iRow, int iCol)
    {
        int i = 0; int j = 0;
        
        for(i = 1; i <= iRow; i++)  
        {
            char ch = (char)('A' + i - 1);
            
            for(j = 1; j <= iCol; j++)   
            {
                System.out.print(ch + " ");
            }
            System.out.println();  
        }
    }
}

class Pattern_3
{
    public static void main (String A[])
    {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter number of rows: ");
        int iRow = sc.nextInt();

        System.out.print("Enter number of columns: ");
        int iCol = sc.nextInt();

        Pattern pobj = new Pattern();
        pobj.Pattern(iRow, iCol);
        
    }
}